### PR TITLE
Makes the door opening for ERTs run async

### PR DESCRIPTION
**Pull request by adamsong**
# Github documenting your Pull Request

The doors no longer open slowly, one at a time.

# Changelog

:cl:  
tweak: tweaked ERT doors to open async
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
I liked the slow opening one by one tbh

**Comment by JamieD1** [comment]
> I liked the slow opening one by one tbh

I agree tbh

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -477,13 +477,13 @@
 			//Open the Armory doors
 			if(ertemplate.opendoors)
 				for(var/obj/machinery/door/poddoor/ert/door in GLOB.airlocks)
-					door.open()
+					INVOKE_ASYNC(door, /obj/machinery/door/poddoor.proc/open)
 					CHECK_TICK
```
Comment (alexkar598) [inline code/modules/admin/verbs/one_click_antag.dm]: What's the point of this if you're going to meme with invoke async

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/one_click_antag.dm (hunk 1 above)
Comment (adamsong) [reply code/modules/admin/verbs/one_click_antag.dm]: I do not understand the point of your comment

Comment (alexkar598) [reply code/modules/admin/verbs/one_click_antag.dm]: What is the point of CHECK_TICK here

Comment (adamsong) [reply code/modules/admin/verbs/one_click_antag.dm]: Ah, I thought you were commenting on a different line

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**